docs/operator-manual/server-commands/argocd-server_version.md

```diff
@@ -0,0 +1,45 @@
+## argocd-server version
+
+Print version information
+
+### Synopsis
+
+Print version information
+
+```
+argocd-server version [flags]
+```
+
+### Options
+
+```
+  -h, --help    help for version
+      --short   print just the version number
+```
+
+### Options inherited from parent commands
```

Comment:
Seems odd that version inherits all these seemingly irrelevant parameters. We may want to consider excluding the `version` command from the generated docs (not sure there is a value add to including it), rather than fixing it.